### ISSUE TITLE
PHP language

### ISSUE THREAD

**Issue by 0x000777** [cnojima/vscode-extension-readable-indent#20]
Can you add support for PHP language. In this example, your extension does not format correctly. A space appears between "=" and ">"

```
$PARAMS = array(
	'PARAM' => 1,
	'PARAM' => "value",
	'PARAM' => 1,
	'PARAM' => "value",
	'PARAM' => 1,
	'PARAM' => "value",
);
```

**Comment by cnojima** [comment]
sure, I should have a few cycles today.

**Comment by cnojima** [comment]
Implemented in publish version 1.2.0.
